kwikapi/api.py

```diff
@@ -508,7 +514,8 @@ def handle_request(self, request):
 
         except Exception as e:
             m = self._handle_exception(request, e)
-            response.write(dict(success=False, message=m), protocol)
+            m['success'] = False
+            response.write(m, protocol)
```

Comment:
@pavan-maddula  Please post an example of response format is changed based on this change

Reply:
when raising exception user can pass a dictionary as a parameter.
Example:
`raise Exception({'message':self.error_message, 'code':103})`

API response using existing kwikapi implementation:
```
{
  "success": false,
  "message": "[(adj_hp100.servers.nferx.com:9105) Exception: {'message': \"No adjacency for '0'\", 'code': 103}]"
}
```
API response of latest implimentation:
```
{
  "code": 103,
  "message": "No adjacency for '0'",
  "success": false,
  "error": "[(adj_hp100.servers.nferx.com:9105) Exception]"
}
```